<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optimize</class>
 <widget class="QWidget" name="optimize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushMax">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>8</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushMin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>8</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>701</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushShowMax">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>33</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Maxima</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushShowMin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>33</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Minima</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTol">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost Tolerance</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkRead">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Live Read Out</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precision</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinPrec">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinMin">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Domain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>15</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>15</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinMax">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variables</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinVar">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
